lib/IRGen/IRGenModule.cpp

```diff
@@ -169,6 +169,18 @@ IRGenModule::IRGenModule(IRGenerator &irgen,
   Int8PtrPtrTy = Int8PtrTy->getPointerTo(0);
   SizeTy = DataLayout.getIntPtrType(getLLVMContext(), /*addrspace*/ 0);
 
+  // For the relative address type, we want to use the int32 bit type
+  // on most architectures, e.g. x86_64, because it produces valid
+  // fixups/relocations. The exception is 16-bit architectures,
+  // so we shorten the relative address type there.
+  if (SizeTy->getBitWidth()<32) {
```

Comment:
Style nit: we generally put spaces around binary operators.

Reply:
OK, smoke tests passed so I won't fix it now. I'm going to do a PR for https://forums.swift.org/t/is-this-builtin-function-now-obsolete/18244/14 over the weekend.  I'll put a whitespace fix in that PR.